src/apps/answers/service.py

```diff
@@ -1922,9 +1923,38 @@ async def is_flow_finished(self, submit_id: uuid.UUID, answer_id: uuid.UUID) ->
         return self._is_activity_last_in_flow(applet_full, activity_id, flow_id)
 
     async def create_report(
```

Comment:
I'm not really sure where to put this comment, but based on my previous comments the implementation of this function will need to change. I just wanted to be explicit about it.

We're currently sending an applet object of type `AppletHistoryFull` to the report server, but it doesn't include the subscales. We should update that object to include the subscales for each activity (as well as the new report score properties, which are already being sent). We don't need to explicitly send `scoringType` or `subscaleTableData`